docs/animations/Fade.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Fade animation behavior
 author: nmetulev
-ms.date: 08/20/2017
 description: The Fade animation behavior fades objects, in and out, over time and delay. It can be used along side other animations directly through XAML or code
 keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, fade, fade animation
 ---
 
 # Fade
 
-The Fade animation fades objects, in and out, over time. Fade animation is applied to all the XAML elements in its parent control/panel. Fade animation doesn't affect the functionality of the control.
+The [Fade animation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.animations.animationextensions) fades objects, in and out, over time. Fade animation is applied to all the XAML elements in its parent control/panel. Fade animation doesn't affect the functionality of the control.
```

Comment:
Possible wrong link or intended to use with id #Microsoft_Toolkit_Uwp_UI_Animations_AnimationExtensions_Fade_